development/_static/css/phpbb.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .wy-table-responsive {
-    float: left;
+    width: 100%;
```

Comment:
Actually needs to be `.wy-table-responsive table` for the table to be full width/responsive.